### DISCUSSION TITLE
using more climates to achieve the result

### DISCUSSION THREAD

**Discussion by jebeke65** [jmcollin78/versatile_thermostat#349] [General]
In the setup, it's possible to link multiple (max 4) climates to achieve the wanted result.
I have e.g. 1 airco and 2 radiators in 1 room. So I want to have these 3 to control the temperature I want.
The airco heatpump is the cheapest one (because of my solar panels and batteries), although the others also are needed as it's quite a big room that has to be controlled.

1. How can I prioritise the usage of the airco heatpump and still have the other 2 radiators help to get the temperature up and running ?
I have 3 possible temperature sensors (each radiator and heat pump have one, and there is also a 4th one).

1. How can I prioritise the usage of the airco heatpump and still have the other 2 radiators help to get the temperature up and running ?
I have 3 possible temperature sensors (each radiator and heat pump have one, and there is also a 4th one).
2. Want I currently do is e.g. set the airco heatpump to 21 degrees and the radiators to 19 degrees (as kind of helping to get the room temperature rising).

**Comment by jmcollin78** [comment]
Hello,

Quick response: 
1. no,
2. no,
3. yes but you need 2 VTherms to do that.

I understand 1. and 2., it have already been claimed but it is not planned yet to develop this feature.
